tests/v2/actions/kubeapi/ingresses/create.go

```diff
@@ -0,0 +1,42 @@
+package ingresses
+
+import (
+	"context"
+
+	"github.com/rancher/shepherd/clients/rancher"
+	"github.com/rancher/shepherd/extensions/unstructured"
+	"github.com/rancher/shepherd/pkg/api/scheme"
+	networkingv1 "k8s.io/api/networking/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// CreateIngress is a helper function that uses the dynamic client to create an ingress on a namespace for a specific cluster.
+func CreateIngress(client *rancher.Client, clusterID, ingressName, namespace string, ingressSpec *networkingv1.IngressSpec) (*networkingv1.Ingress, error) {
```

Comment:
extension?

Reply:
I don't think any call that is just really being used to convert to an unstructured should be an action.